README.md

```diff
@@ -10,7 +10,8 @@ of containerd's `ctr` tool (`ctr-enc') with support for encrypting and decryptin
 
 # Usage
 
-`imgcrypt` requires containerd 1.3 or later.
+`imgcrypt` requires containerd 1.3 or later. Containerd 1.4 or later is required when used with Kubernetes.
+For configuration instructions for kubernetes, please consult the [CRI decryption document](https://github.com/containerd/cri/blob/master/docs/decryption.md).
```

Comment:
The CRI doc was recently moved to https://github.com/containerd/containerd/blob/master/docs/decryption.md

Reply:
Thanks for the heads up!